pkg/crc/config/validations.go

```diff
@@ -127,3 +127,14 @@ func validatePreset(value interface{}) (bool, string) {
 	}
 	return true, ""
 }
+
+func ValidatePort(value interface{}) (bool, string) {
+	port, err := cast.ToUintE(value)
+	if err != nil {
+		return false, "Requires integer value in range of 1024-65535"
```

Comment:
below 1024 is now allowed?

Reply:
I would avoid using something below 1024 because most of those ports are used by some services.